tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -357,6 +323,19 @@ def system_default_switchport(agent, state=false,
   end
 end
 
+# Helper for checking/setting 'system default switchport'
+def config_system_default_switchport?(tests, id)
```

Comment:
I did. They're not identical, they do the actual config change while my new methods are just used to check if the changes are actually necessary. Also, some callers are calling those direct, in my case I'm just front-ending them to avoid having to set the state for every test case.

I resisted overloading the existing methods but I can add my logic to them if you think that would be cleaner.